src/components/pages/search-page/SearchPage.jsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react'
+import LabelledInput from '../../labelled-input/LabelledInput'
+import './SearchPage.css'
+import Button from '../../button/Button'
 
 const SearchPage = () => {
-  return <div />
+  const centered = true
+  return (
+    <>
+      <form action="" className="search-page-form">
+        <LabelledInput
+          label="ID заказа"
+          labelClassName="search-page-form-fieldset"
+          centered={centered}
+        />
+        <LabelledInput
+          label="Телефон"
+          labelClassName="search-page-form-fieldset"
+          centered={centered}
+        />
+        <LabelledInput
+          label="Город"
+          labelClassName="search-page-form-fieldset"
+          centered={centered}
+        />
+        <LabelledInput
+          label="Улица"
+          labelClassName="search-page-form-fieldset"
+          centered={centered}
+        />
+        <LabelledInput
+          label="Дом"
+          labelClassName="search-page-form-fieldset"
```

Comment:
давай уменьшим название этого класса
```suggestion
          labelClassName="search-page-form-field"
```